acceptance/tests/language/class_inheritance.rb

```diff
@@ -0,0 +1,19 @@
+test_name 'C14943: Class inheritance works correctly' do
+
+  agents.each do |agent|
+    test_manifest = <<MANIFEST
+      class bar { notice("This is class $name") }
+      class foo::bar { notice("This is class $name") }
+      class foo inherits bar { notice("This is class $name") }
+
+      include foo
+MANIFEST
+
+    apply_manifest_on agent, test_manifest do
+      assert_match(/Scope\(Class\[Bar\]\): This is class bar/, stdout, 'did not find class bar')
+      assert_match(/Scope\(Class\[Foo\]\): This is class foo/, stdout, 'did not find class foo')
+      refute_match(/This is class foo::bar/, stdout, 'should not have found class foo::bar')
```

Comment:
Will these matchers still be evaluated if applying the manifest fails and returns non-zero exit code?